Kitodo/src/test/java/org/kitodo/NewspaperCourse.java

```diff
@@ -0,0 +1,181 @@
+/*
+ * (c) Kitodo. Key to digital objects e. V. <contact@kitodo.org>
+ *
+ * This file is part of the Kitodo project.
+ *
+ * It is licensed under GNU General Public License version 3 or later.
+ *
+ * For the full copyright and license information, please read the
+ * GPL3-License.txt file that was distributed with this source code.
+ */
+
+package org.kitodo;
+
+import java.util.Objects;
+
+import org.joda.time.LocalDate;
+import org.kitodo.production.model.bibliography.course.Block;
+import org.kitodo.production.model.bibliography.course.Course;
+import org.kitodo.production.model.bibliography.course.Issue;
+
+public class NewspaperCourse {
+    private static Course course;
+
+    /**
+     * Returns a course of appearance.
+     *
+     * @return a course of appearance
+     */
+    public static Course getCourse() {
+        if (Objects.isNull(course)) {
+            course = createCourse();
+        }
+        return course;
+    }
+
+    private static Course createCourse() {
+        Course course = new Course();
+        addFirstBlock(course);
+        addSecondBlock(course);
```

Comment:
Is there a reason to add more than one block? As there are no assertions, i'm not quite sure, what is testet. Seems to me the first block is enough to cover a lot of cases.
Due to the fact, that test execution takes more than 20 minutes, I think we should limit the number of processes to a minimum